bittensor/_dataset/dataset_impl.py

```diff
@@ -42,10 +42,12 @@ class Dataset():
     def __init__(self):
         
         # Used to retrieve directory contentx
-        self.file_get = 'http://ipfs2.opentensor.ai/api/v0/object/get'
-        self.pin_get = 'http://ipfs2.opentensor.ai/api/v0/pin/ls'
+        self.cat = 'http://ipfs.opentensor.ai:5001/api/v0/cat' 
+        self.node_get = 'http://ipfs.opentensor.ai:5001/api/v0/object/get'
+        self.mountain_hash = 'QmSdDg6V9dgpdAFtActs75Qfc36qJtm9y8a7yrQ1rHm7ZX'
```

Comment:
We shouldn't expose ports. Let me know if you want to set up nginx on ipfs.opentensor.ai